environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -284,4 +284,18 @@ void updateEnvironmentLoadBalancersByCrn(@ValidCrn(resource = CrnResourceDescrip
     @Produces(MediaType.APPLICATION_JSON)
     ObjectStorageValidateResponse validateCloudStorage(@Valid EnvironmentCloudStorageValidationRequest environmentCloudStorageValidationRequest);
 
+    @PUT
+    @Path("/name/{name}/upgrade_ccm")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.UPGRADE_CCM, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
+            nickname = "upgradeCcmByEnvironmentNameV1")
```

Comment:
Note: It is certainly not a fault of this change, but the usage / omission of the `nickname` suffix `V1` seems to be quite inconsistent in this IF.

Reply:
I just leave as it is